core/src/test/java/org/apache/brooklyn/location/ssh/SshMachineLocationPerformanceTest.java

```diff
@@ -79,7 +78,7 @@ public void testConsecutiveSmallCommands() throws Exception {
     }
 
     // Mimics SshSensorAdapter's polling
-    @Test(groups = {"Integration"})
+    @Test(groups={"Integration", "Performance"})
```

Comment:
This does get run if you do `mvn test -Dtest=SshMachineLocationPerformanceTest -PPerformance`, and gets excluded if you do `-PIntegration`. This is because "Integration" is not excluded from the "Performance" profile, but "Performance" *is* excluded from the "Integration" profile.

I'll comment more on that against your comment in `parent/pom.xml`.